tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md

```diff
@@ -149,5 +149,10 @@ The expected failures in this file are from features in the owncloud/ocis repo.
 - [apiLocks/lockFiles.feature:180](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiLocks/lockFiles.feature#L180)
 - [apiLocks/lockFiles.feature:181](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiLocks/lockFiles.feature#L181)
 
+### [Error 500 when user try to unlock project space file](https://github.com/owncloud/ocis/issues/7696)
+
+- [apiLocks/unlockFiles.feature:21](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiLocks/unlockFiles.feature#L21)
+- [apiLocks/unlockFiles.feature:22](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiLocks/unlockFiles.feature#L22)
```

Comment:
@ScharfViktor do we wait for fix or just keep it here for now?

Reply:
let keep it here. I ask fix it on the next week